### PR TITLE
[NO-JIRA] Greenkeeper ignore libs

### PR DESCRIPTION
**Pull request by tiagoengel**
Ignoring some libs we don't want greenkeeper to update

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,5 +108,16 @@
...
     "react-native-linear-gradient": "react-native-community/react-native-linear-gradient#20928ce2ac8038e8545c7f6c3faab05f866deabd",
     "react-test-renderer": "16.6.1"
+  },
+  "greenkeeper": {
+    "ignore": [
+      "babel-loader",
+      "react",
+      "react-dom",
+      "react-test-renderer",
+      "webpack",
```
[the remark this answers is not in the document]
Comment (tiagoengel) [reply package.json]: Oh true, forgot all of this is inside the rn cli

File: packages/react-native-bpk-component-animate-height/package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "react-native-bpk-component-animate-height",
-	"version": "2.0.57",
+	"version": "2.0.58",
```
Comment (georgegillams) [inline packages/react-native-bpk-component-animate-height/package-lock.json]: How come this has changed? They don't seem to be out of date on Master?

Comment (georgegillams) [reply packages/react-native-bpk-component-animate-height/package-lock.json]: My bad - they are outdated on master :)

Comment (tiagoengel) [reply packages/react-native-bpk-component-animate-height/package-lock.json]: they are https://github.com/Skyscanner/backpack-react-native/blob/master/packages/react-native-bpk-component-animate-height/package-lock.json.

It's prob the same lerna problem we have in the main repo

Comment (georgegillams) [reply packages/react-native-bpk-component-animate-height/package-lock.json]: Cool. I've fixed them on master just to make the commit history more interpretable!

**Pull request merged**